<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
  * Released as open source by Gabriel Caudrelier
  *
  * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
  *
  * https://github.com/metrodango/pip3line
  *
  * Released under BSD 3-Clause see LICENSE for more information
</comment>
 <class>AnalyseDialog</class>
 <widget class="QDialog" name="AnalyseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysing sample</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="modulesGroupBox">
     <property name="title">
      <string>Modules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="charsetPushButton">
        <property name="text">
         <string>Charset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="charFrequencyPushButton">
        <property name="text">
         <string>Char frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="guessHashpushButton">
        <property name="text">
         <string>Guess hash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="inputDecodingGroupBox">
        <property name="title">
         <string>Hash input is encoded as:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="base64RadioButton">
           <property name="text">
            <string>Base64</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="hexadecimalRadioButton">
           <property name="text">
            <string>hexadecimal</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>One hash per line
Any errors during decoding is ignored</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="clearInputPushButton">
        <property name="text">
         <string>Clear input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="inputPlainTextEdit"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="resultsPlainTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cleanPushButton">
        <property name="text">
         <string>Clear results</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionImportFile">
   <property name="text">
    <string>importFile</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>launchAnalysis()</slot>
  <slot>onInputRightClick(QPoint)</slot>
 </slots>
</ui>
